packages/core/src/lib/compare.ts

```diff
@@ -29,7 +39,14 @@ export async function compareReportFiles(
 
   const reportsDiff = compareReports(reports);
 
-  await writeFile(outputPath, JSON.stringify(reportsDiff, null, 2));
+  return Promise.all(
+    format.map(async fmt => {
+      const outputPath = join(outputDir, `${filename}-diff.${fmt}`);
+      const content = reportsDiffToFileContent(reportsDiff, fmt);
+      await writeFile(outputPath, content);
```

Comment:
Shouldn't we make sure the `outputPath` exists like in `persist` in `core`?

Reply:
Good point. Updated.